docker/ssl/start.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+echo "Setting up Let's Encrypt SSL for IRI using nginx proxy_pass"
+
+while getopts 'e:d:p:t' flag; do
+  case "${flag}" in
+    e) EMAIL="${OPTARG}" ;;
+    d) DOMAIN="${OPTARG}" ;;
+    p) PORT="${OPTARG}" ;;
+    t) TOS="${OPTARG}" ;;
+  esac
+done
+
+# Email
+if [[ -z "$EMAIL" ]]; then
+  echo "Please enter your email (used by certbot for urgent renewal and security notices):"
+  read EMAIL
+  if [[ -z "$EMAIL" ]]; then
+    echo "Error: You must enter an email"
+    exit 1
+  fi
+fi
+
+# Domain
+if [[ -z "$DOMAIN" ]]; then
+  echo "Please enter a domain that points to this system's IP:"
+  read DOMAIN
+  if [[ -z "$DOMAIN" ]]; then
+    echo "Error: You must enter a domain"
+    exit 1
+  fi
+fi
+
+# Port
+if [[ -z "$PORT" ]]; then
+  echo "Please enter your desired IRI API port:"
```

Comment:
Do we still need this since we do not expose the API port directly anymore?

Reply:
This is still needed in order to tell `nginx` which port to forward requests to. It's also necessary for the IRI docker image:
> Run the official iotaledger/iri container, passing the mandatory -p option:
> `docker run iotaledger/iri:v1.5.4 -p 14265`

https://github.com/iotaledger/iri/blob/dev/DOCKER.md

Reply:
I am just saying we can always just go for the default 14265 one. But it's fine to me anyway.